tests/integ/sagemaker/jumpstart/script_mode_class/test_transfer_learning.py

```diff
@@ -0,0 +1,132 @@
+    EndpointInvoker,
+    get_sm_session,
+    get_training_dataset_for_model_and_version,
+)
+import pandas as pd
+
+
+def test_jumpstart_transfer_learning_estimator_class(setup):
+
+    model_id, model_version = "huggingface-spc-bert-base-cased", "1.0.0"
+    training_instance_type = "ml.p3.2xlarge"
+    inference_instance_type = "ml.p2.xlarge"
+    instance_count = 1
+
+    print("Starting training...")
+
+    image_uri = image_uris.retrieve(
+        region=None,
+        framework=None,
+        image_scope="training",
+        model_id=model_id,
+        model_version=model_version,
+        instance_type=training_instance_type,
+    )
+
+    script_uri = script_uris.retrieve(
+        model_id=model_id, model_version=model_version, script_scope="training"
+    )
+
+    model_uri = model_uris.retrieve(
+        model_id=model_id, model_version=model_version, model_scope="training"
+    )
+
+    default_hyperparameters = hyperparameters.retrieve_default(
+        model_id=model_id,
+        model_version=model_version,
+    )
+
+    default_hyperparameters["epochs"] = "1"
+
+    estimator = Estimator(
+        image_uri=image_uri,
+        source_dir=script_uri,
+        entry_point=TRAINING_ENTRYPOINT_SCRIPT_NAME,
+        role=get_sm_session().get_caller_identity_arn(),
+        sagemaker_session=get_sm_session(),
+        enable_network_isolation=True,
+        hyperparameters=default_hyperparameters,
+        tags=[{"Key": JUMPSTART_TAG, "Value": os.environ[ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID]}],
+        instance_count=instance_count,
+        instance_type=training_instance_type,
+    )
+
+    estimator.fit(
+        {
+            "training": f"s3://{get_jumpstart_content_bucket(JUMPSTART_DEFAULT_REGION_NAME)}/"
+            f"{get_training_dataset_for_model_and_version(model_id, model_version)}",
+            "model": model_uri,
+        }
+    )
+
+    print("Starting inference...")
+
+    image_uri = image_uris.retrieve(
+        region=None,
+        framework=None,
+        image_scope="inference",
+        model_id=model_id,
+        model_version=model_version,
+        instance_type=inference_instance_type,
+    )
+
+    script_uri = script_uris.retrieve(
+        model_id=model_id, model_version=model_version, script_scope="inference"
+    )
+
+    model_uri = model_uris.retrieve(
+        model_id=model_id, model_version=model_version, model_scope="inference"
+    )
+
+    endpoint_name = name_from_base(f"{model_id}-transfer-learning")
+
+    estimator.deploy(
+        initial_instance_count=instance_count,
+        instance_type=inference_instance_type,
+        entry_point=INFERENCE_ENTRYPOINT_SCRIPT_NAME,
+        image_uri=image_uri,
+        source_dir=script_uri,
+        endpoint_name=endpoint_name,
+    )
+
+    endpoint_invoker = EndpointInvoker(
+        endpoint_name=endpoint_name,
+    )
+
+    response = endpoint_invoker.invoke_spc_endpoint(["hello", "world"])
```

Comment:
careful, we might change the response type, this might be introducing coupling. I know that you are pinning the model version, but still. How about just checking that response is not `None`?